src/Resources/assets/controllers/input-mask_controller.js

```diff
@@ -0,0 +1,27 @@
+import { Controller } from "@hotwired/stimulus"
+import IMask from "imask"
+export default class extends Controller {
+    static values = {
+        mask: String,
+        scale: { type: Number, default: 2 },
+        radix: { type: String, default: '.' },
+        thousandsSeparator: { type: String, default: '\'' },
+        normalizeZeros: { type: Boolean, default: false },
+    };
+    connect() {
+        const typeMapping = {
+            'Number': Number,
+        };
```

Comment:
Nit: Just from an ergonomic perspective, this very static and can live outside the class.